### PR TITLE
BUG-1886087 adding HostStatusResettingPendingUserAction and HostStatu…

### PR DESCRIPTION
**Pull request by tsorya**
…sDiscovery to filtered state for ntp validation

### CODE REVIEW

File: internal/cluster/validator.go [hunk 1]
```diff
@@ -406,7 +406,9 @@ func (v *clusterValidator) isNtpServerConfigured(c *clusterPreprocessContext) va
 			v.log.WithError(err).Warnf("Illegal inventory for host %s", h.ID.String())
 			continue
 		}
-		if inventory.Timestamp == 0 || *h.Status == models.HostStatusDisconnected || *h.Status == models.HostStatusDisabled {
+		if inventory.Timestamp == 0 || *h.Status == models.HostStatusDisconnected ||
+			*h.Status == models.HostStatusDisabled || *h.Status == models.HostStatusResettingPendingUserAction ||
+			*h.Status == models.HostStatusDiscovering {
```
Comment (gamli75) [inline internal/cluster/validator.go:411]: why you added discovering? I think it's wrong - you should validate it in all states that we are running inventory commands: https://github.com/openshift/assisted-service/blob/master/internal/host/instructionmanager.go#L75

**Review by ronniel1 — APPROVED** [review]
/approve
/lgtm

File: internal/cluster/validator.go (hunk 1 above)
Comment (tsorya) [reply internal/cluster/validator.go:411]: discussed and agreed it is ok

**Pull request merged**